gpii/configs/mocks/gpii.config.untrusted.development.mock.windows.txt

```diff
@@ -0,0 +1,23 @@
+gpii.config.cloudBased.development.all.local.mock.windows.json
+=================================================
+
+A variant of gpii.cloudBased.development.all.local which includes mock definitions for all builtin settings handlers
+supported on the Windows platform. This allows any preference set to be logged in from a bare universal repo without
+crashing the system.
+
+This configuration runs the entire system as a cloudBased flowmanager, but with everything locally and in development mode.
+
+This mean that:
+* The storage for user preferences is on the local filesystem (as opposed to couchDB servers/remote URLs)
+* The storage for the authentication data is stored locally in a pouchDB
+* The following components are running on the local machine:
+** flowManager
+** preferencesServer
+** OntologyHandler
+** rawPreferencesServer (reading preferences from file)
+** solutionsRegistry (reading registry from file)
+** canopyMatchMaker
+** matchMakerFramework
+** lifecycleManager
```

Comment:
Why would the lifecycle manager be in a cloud based flowmanager? Isn't this only ever relevant for a local machine?

Reply:
Well, that's a very good point - I've tried moving to flowManager.local and nothing seemed to blow up - remarkable!